packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -677,6 +700,15 @@ class WSIViewport extends Viewport {
     this.getView()?.setZoom(zoom);
   }
 
+  /**
+   * Returns the list of image Ids for the viewport.  Currently only
+   * returns the first/primary image id.
+   * @returns list of strings for image Ids
+   */
+  public getImageIds = (): Array<string> => {
```

Comment:
It is strange, since we have setDataIds and this is getImageIds, maybe we keep the setWSI?

Reply:
This is getting called in some of the navigation examples for wheel mouse.  I was just adding it to prevent exceptions getting thrown.